### DISCUSSION TITLE
Minor code cleanup suggestion

### DISCUSSION THREAD

**Discussion by DE-cr** [fredlcore/BSB-LAN#650] [General]
1. Shouldn't check-doubles.pl go into the scripts sub-directory?
2. Is json-api-version.h necessary? It only holds two constant defines, which are used just once in BSB_LAN.ino

**Comment by fredlcore** [comment]
That's correct, moved the file. 
json-api-version.h is used in a similar way as bsb-version.h, i.e. to quickly change the API version without having to touch the main files.
